jobs/user_add/templates/pre-start.sh.erb

```diff
@@ -42,13 +45,8 @@ set -ex
     usermod -s <%=user_hash['shell']%> <%=user%>
   <% end %>
 
-  if grep -q 'bosh_sshers' /etc/group; then
-    <% if sudo %>
-      usermod -G vcap,admin,bosh_sudoers,bosh_sshers <%=user%>
-    <% else %>
-      usermod -G vcap,bosh_sshers <%=user%>
-    <% end %>
-  fi
+  echo <%= groups.join(' ') %> | xargs -n 1 groupadd -f
```

Comment:
This is a stylistic preference, but I'd prefer if we used an `erb` loop here rather than xargs. Something like:
```
<% groups.each do |group| %>
  groupadd -f <%= group %>
<% end %>
```